types/evidence.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	// MaxEvidenceBytes is a maximum size of any evidence (including amino overhead).
-	MaxEvidenceBytes int64 = 484
+	MaxEvidenceBytes int64 = 444
```

Comment:
Why did you change this?

Reply:
the size of duplicatevoteEvidence shrunk. 

I should add table driven tests for all types of evidence for max evidenceBytes

Reply:
Does the DuplicateVoteEvidence dictate the evidence size? Seems unfair on the other types of evidence

Reply:
all the new types exceed the max size

```
   TestMaxEvidenceBytes: evidence_test.go:147: 
        	Error Trace:	evidence_test.go:147
        	Error:      	Not equal: 
        	            	expected: int64(444)
        	            	actual  : int(640)
        	Test:       	TestMaxEvidenceBytes
        	Messages:   	LunaticValidatorEvidence
    TestMaxEvidenceBytes: evidence_test.go:147: 
        	Error Trace:	evidence_test.go:147
        	Error:      	Not equal: 
        	            	expected: int64(444)
        	            	actual  : int(649)
        	Test:       	TestMaxEvidenceBytes
        	Messages:   	PhantomValidatorEvidence
    TestMaxEvidenceBytes: evidence_test.go:147: 
        	Error Trace:	evidence_test.go:147
        	Error:      	Not equal: 
        	            	expected: int64(444)
        	            	actual  : int(3124)
        	Test:       	TestMaxEvidenceBytes
        	Messages:   	ConflictingHeadersEvidence
```

I'm not sure how much to bump it as 3124 seems like quite a bit

Reply:
The conflicting headers doesn't get submitted to the block so I wonder if we count it or not

Reply:
Just leave it for now and maybe we should look into it with a separate issue - I'm not sure how important the number is

Reply:
commented it out for now